python/py-aiopg/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-aiopg
+version             1.1.0
+revision            0
+categories-append   devel
+
+license             MIT
+maintainers         nomaintainer
+platforms           darwin
+supported_archs     noarch
+
+description         Postgres integration with asyncio.
+
+long_description    aiopg is a library for accessing a PostgreSQL \
+                    database from the asyncio (PEP-3156/tulip) \
+                    framework. It wraps asynchronous features of the \
+                    Psycopg database driver.
+
+homepage            https://aiopg.readthedocs.io/
+
+checksums           rmd160  140d8fac33c16bd4ba0e0732a3297ae8f1e49a32 \
+                    sha256  7583b2a5477072c41e83ffb252f27ec3188abf281dfa88645d389be4cd0aa098 \
+                    size    33307
+
+python.versions     37 38 39
+
+if {${name} ne ${subport}} {
+    depends_lib-append \
+                    port:py${python.version}-setuptools
```

Comment:
Isn't this also dependent on `psycopg2` or `psycopg2-binary`?

https://github.com/aio-libs/aiopg/blob/b5880d25eb402959723b6979d4840ae42ff812ea/setup.py#L6

Reply:
Thanks, done. Also, MacPorts `py*-psycopg2` has the binary build.